Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -315,6 +318,87 @@ - (void)testAllMutationBatchesAffectingDocumentKey {
   });
 }
 
+- (void)testAllMutationBatchesAffectingDocumentKeys {
+  if ([self isTestBaseClass]) return;
+
+  self.persistence.run("testAllMutationBatchesAffectingDocumentKey", [&]() {
+    NSArray<FSTMutation *> *mutations = @[
+      FSTTestSetMutation(@"fob/bar",
```

Comment:
Ultranit: could you replace the `b` in `fob` with a letter that has less resemblance to an `o`, say, `x` or `k` or `y`? It took me three passes to finally spot the difference between this element and the next one.